<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookingConfirmation</class>
 <widget class="QDialog" name="BookingConfirmation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="EventNameLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Event Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="DateLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date </string>
   </property>
  </widget>
  <widget class="QLabel" name="LocationLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
  </widget>
  <widget class="QLabel" name="NoOfTicketsLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Tickets</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmBookingButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm Booking</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberoftickets">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
